### DISCUSSION TITLE
Image in a list in a popup doesn't show

### DISCUSSION THREAD

**Discussion by etshy** [fomantic/Fomantic-UI#449] [Elements (button, image, label, loader, ...)] [answered]
# Bug Report

## Steps to reproduce
1. make a popup
2. make a list inside with image
3. image doesn't show

## Expected result
Image is showing

## Actual result
Image isn't showing

## Testcase
[JSFiddle](https://jsfiddle.net/y3fn21wk/2/)

## Version
2.7.1

Adding a minimum size could fix this but... which size to put ?

**Comment by ColinFrick** [comment, marked as answer]
Don't wrap the image in a div element and add one of the image size variations:
https://jsfiddle.net/o61dkwsn/

https://fomantic-ui.com/elements/list.html#image
https://fomantic-ui.com/elements/image.html#size

**Comment by etshy** [comment]
Oh true, sorry...

I first made an `items` and changhed it to a `list` after but didn't change the html structure...

My bad, and Thanks.
